homeassistant/util/color.py

```diff
@@ -426,7 +426,7 @@ def color_rgb_to_rgbw(r: int, g: int, b: int) -> tuple[int, int, int, int]:
 
     # Match the output maximum value to the input. This ensures the full
     # channel range is used.
-    return _match_max_scale((r, g, b), rgbw)  # type: ignore
+    return match_max_scale((r, g, b), rgbw)  # type: ignore[return-value]
```

Comment:
Shall we add @overloads for the used tuple length variations (3, 4 and 5 x `int`)? or better keep the ignores?

Reply:
ignore is fine.